### ISSUE TITLE
Shrink text on overflow fails in MS PowerPoint

### ISSUE THREAD

**Issue by vineetovhal** [gitbrent/PptxGenJS#544]
After researching and looking at different issues related to Shrink text on overflow, I found that even after using <shrinkText: true>, MS PowerPoint does not shrink the text. The interesting thing is when you open the downloaded PPT and look at the format shape options, "Shrink text on overflow" is pre-selected as the value is set to be true, however, the text does not shrink. One needs to de-select it and select it back within MS PowerPoint which then shrinks the text. 
This does not happen when it comes to using OpenOffice and LibreOffice PPT. 

Is there a any other way to make shrinkText function work for MS PowerPoint? 

Thanks

**Comment by gitbrent** [comment]
[Closing due to age]

**Comment by idris** [comment]
@gitbrent I'm running into this same issue.  Any ideas?

**Comment by gitbrent** [comment]
@idris - no, like the desc says, the XML is fine...?

**Comment by vnom2112** [comment]
@gitbrent I am also running into this issue.  I saw the same behavior as the OP when viewing the properties of the textbox.  I did compare the XML and it is not the same.  I downloaded the generated PPT and compared it's xml with the same PPT except that I deselected "shrink to fit" then reselected and saved.  Here are the results:

![image](https://user-images.githubusercontent.com/4922708/92531326-ad982e00-f1f3-11ea-8da9-8b745f47f74c.png)

**Comment by jdimmerman** [comment]
I am seeing the same behavior. Was anyone able to solve this? It looks like poweropint saves the fit parameters into the XML at time of save, rather than just taking into account the normAutofit property itself.

**Comment by Jaydemir89** [comment]
@jdimmerman @gitbrent I can pretty much confirm what the OP said, its a Powerpoint specific problem, as it works fine on LibreOffice. Simply saving and reopening the file didn't do anything, I had to directly interact with the textbox (typing a character snapped everything back to fitting into the textbox). 

If I opened the file in LibreOffice, saved it, then opened in Powerpoint, everything remained in the textbox but readjusted itself anyway when I started typing. If It's just a matter of getting everything to fit in the text boxes, just open the generated file with LibreOffice and save as PPTX as a workaround.
